python/ray/tune/tests/test_multinode_sync.py

```diff
@@ -84,7 +84,7 @@ def testAutoscalingNewNode(self):
         """Test that newly added nodes from autoscaling are not stale."""
         self.cluster.update_config(
             {
-                "provider": {"head_resources": {"CPU": 4, "GPU": 0}},
```

Comment:
# Making `head_resources` 2 CPU will forces this test to scale up at least
# one worker node. Otherwise, this test may pass without scaling up at all by 
# scheduling task on the head node